Documentation/special-services/pod-mutator.md

```diff
@@ -0,0 +1,74 @@
+# PodMutator Service
```

Comment:
I believe `PodMutator` will be hard to understand for a casual observer, I'm inclined to use `AppManager`